core/src/components/title/title.ios.scss

```diff
@@ -14,7 +14,7 @@
 
   transform: translateZ(0);
 
-  font-size: 17px;
+  font-size: dynamic-font-min(1, 17px);
```

Comment:
This is going to set a minimum font size. Did you mean to use `dynamic-font-max` instead?